src/main/java/net/fabricmc/meta/web/EndpointsV2.java

```diff
@@ -34,145 +34,170 @@
 @SuppressWarnings("Duplicates")
 public class EndpointsV2 {
 
-	public static void setup() {
+    public static void setup() {
```

Comment:
Sorry about this diff, I must have pressed Ctrl Alt L in Idea and auto formatted everything